src/lib/support/tests/BUILD.gn

```diff
@@ -60,7 +60,10 @@ chip_test_suite("tests") {
     test_sources += [ "TestCHIPArgParser.cpp" ]
   }
 
-  cflags = [ "-Wconversion" ]
+  cflags = [
+    "-Wconversion",
+    "-Wno-error=uninitialized",
```

Comment:
At the very least this needs a comment explaining why this is here.

Reply:
Added comment